### ISSUE TITLE
Allow for customization of the S3 Bucket

### ISSUE THREAD

**Issue by mkielar** [Kotaimen/awscfncli#57]
In [#11](https://github.com/Kotaimen/awscfncli/issues/11), you mentioned that in 2.x it will be possible to customize the S3 bucket used by cloudformation package. Yet I cannot find the option to do so, and the code section responsible for the bucket name doesn't look promising either.

Can I please have a way of customizing the S3 bucket?
I'm currently on 0.5x using cloudformation package as a separate call, and then running `cfn` to get he log tail, and I'm starting migration to 2.x, but this is what is blocking me currently from fully migrating to `sync` command.

**Comment by Kotaimen** [comment]
Hi @mkielar:
Yes, this feature is already implemented in 2.0 and later, you can overwrite the bucket using `ArtifactStore` parameter:
```yaml
Stages:

  Develop:
    ApiBackend-Develop:
    Template: template.yaml
    Capabilities: [CAPABILITY_IAM]
    Package: true
    Region: us-east-1
    ArtifactStore: awscfncli-889515947644-custom-artifact-bucket
```

Note `awscfncli` won't create custom bucket automatically and the bucket must be in the same region with your stack.

**Comment by mkielar** [comment]
Oh, thanks. The buckets are already there with encryption, short retention policy, etc, which is why I want to reuse them. You may want to improve the docs, as this feature seems to be missing, and this issue can be closed.

By the way, `sync` works beautifully, thanks for the great tool.

**Comment by mkielar** [comment]
I'm reopening this, as I will need an option to pass the bucket name as a command line parameter, similar to what was possible in 0.5x

This is a case where I have to run exactly same CF Stack on multiple different AWS Accounts, and can only make use of already existing buckets. With current approach, I'd have to keep separate configuration files for each account, and they would only differ by the `ArtifactStore` value.

**Comment by mkielar** [comment]
I'm wondering if you'd consider merging a pull request?

**Comment by Kotaimen** [comment]
@GlieseRay please release the pull request with a hot fix of `2.0.x`.

**Comment by GlieseRay** [comment]
Hi @Marcin, I have released a new 2.0.5 release. Please check.

**Comment by mkielar** [comment]
Works as expected. Thanks!
